### PR TITLE
fix: identify instance by host and documentation

### LINKED ISSUE #154: Fix instance documentation
Hi, the #118 was incomplete and I'd like to add documentation.

### PR DESCRIPTION
**Pull request by manuelstein**
closes #154

## 🚨 Proposed changes

fixes incomplete changes to identify GitLab instances.
adds documentation

## ⚙️ Types of changes

What types of changes does your code introduce? _Put an `x` in the boxes that apply_

-   [ ] New feature (non-breaking change which adds functionality)
-   [X] Bugfix (non-breaking change which fixes an issue)
-   [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
-   [X] Documentation Update (if none of the other choices apply)
-   [ ] Refactor

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -214,7 +214,7 @@ gitlab:
 
 ## Annotations
 
-By default, the plugin automatically shows the project info corresponding to the location of the `catalog.yaml` file. But you could need some time to force another project, you can do it with the annotations `gitlab.com/project-id` or `gitlab.com/project-slug`:
+By default, the plugin automatically shows the project info corresponding to the location of the `catalog.yaml` file. But you could need some time to force another project. The GitLab instance defaults to `gitlab.com` and can use any configured [GitLab integration host](https://backstage.io/docs/integrations/gitlab/locations) with the annotation `gitlab.com/instance`. The project can be set using with the annotations `gitlab.com/project-id` or `gitlab.com/project-slug`:
```
Comment (antoniomuso) [inline README.md:217]: I think it is better not to add the annotation `gitlab.com/instance` in the documentation.

File: README.md [hunk 2]
```diff
@@ -223,8 +223,11 @@ kind: Component
 metadata:
     # ...
     annotations:
+        # Select the GitLab integration by its host name
+        gitlab.com/instance: 'hostname' # defaults to 'gitlab.com'
```
Comment (antoniomuso) [inline README.md:227]: As said in the previous comment.

**Review by antoniomuso — CHANGES_REQUESTED** [review]
Hi @manuelstein, Thank you for the PR. I prefer not to specify the `instance` annotation in the documentation because the processor automatically generates this annotation.

**Comment by manuelstein** [comment]
I see you've fixed the IssuesTable and MergeRequestsTable from this PR 3 weeks ago, which is the crucial part.
Thx, Manuel

**Pull request closed without merging**